pkg/hub/controllers/cluster/reconciler.go

```diff
@@ -272,6 +270,15 @@ func (r *Reconciler) getCluster(ctx context.Context, req reconcile.Request) (*hu
 	return hubCluster, nil
 }
 
+func isEmptyString(nodeIPs []string) bool {
+	for _, nodeIP := range nodeIPs {
+		if nodeIP == "" {
+			return true
+		}
```

Comment:
So if the array is ["10.1.2.2",""]. then this method returns true. is it really what is intended?

Reply:
Yes this is intended. so we are going to range through all the nodeIPs and if we found empty string we will return true.